var/spack/repos/builtin/packages/py-quantities/package.py

```diff
@@ -15,6 +15,7 @@ class PyQuantities(PythonPackage):
 
     license("DOC")
 
```

Comment:
Seems like an unrelated change, could be in a separate PR

Reply:
Can you elaborate? Does one of these packages not support Python 3.11?

Reply:
Hm, just noticed that this package dropped the requirements on Python. I forward-ported this PR from Spack 0.21, where Python was constrained to 3.10: https://github.com/spack/spack/blob/v0.21.1/var/spack/repos/builtin/packages/py-quantities/package.py#L23

I can take this package out of the PR, then.

Reply:
And done.